### ISSUE TITLE
[Feature Request] Hyprbars - hover and active button colors

### ISSUE THREAD

**Issue by LewentalDigital** [hyprwm/hyprland-plugins#176]
Would be nice to have **color settings** for the buttons on **mouse hover** and **active** states

**Comment by suderman** [comment]
Yes please! And inactive states as well, so I can dim or hide buttons on inactive windows.

**Comment by Candyhands** [comment]
Ditto to having different colors for different states. Especially for inactive/active. Having the titlebar being a different color is by far the fastest way for me to tell which window is the active one. Having the inactive windows be slightly transparent is okay. But if one of my inactive windows is a text tile I'm reading notes from, then having that not fully opaque is a bother.

**Comment by mkavanagh-23** [comment]
In addition to color on hover state I'd also add the ability to set button icons on hover
